lms/services/vitalsource/client.py

```diff
@@ -13,11 +16,44 @@ def __init__(self, lti_launch_key, lti_launch_secret):
         :type lti_launch_secret: str
         :raises ValueError: If credentials are invalid
         """
-        if not lti_launch_key or not lti_launch_secret:
-            raise ValueError("VitalSource LTI launch credentials are invalid")
+        if not all([lti_launch_key, lti_launch_secret, api_key]):
+            raise ValueError("VitalSource credentials are invalid")
+
+        self._http_service = http_service
 
         self._lti_launch_key = lti_launch_key
         self._lti_launch_secret = lti_launch_secret
+        self._api_key = api_key
+
+
+    def _api_get(self, endpoint):
```

Comment:
This seems to be all we need as an "API Client"

Reply:
:+1: